ee/codegen/src/generators/sandbox-file.ts

```diff
@@ -0,0 +1,113 @@
+import { python } from "@fern-api/python-ast";
+import { AstNode } from "@fern-api/python-ast/core/AstNode";
+
+import { BasePersistedFile } from "./base-persisted-file";
+
+import type { Inputs } from "./inputs";
+import type { WorkflowFile } from "./workflow";
+
+export declare namespace SandboxFile {
+  interface Args extends BasePersistedFile.Args {
+    workflowFile: WorkflowFile;
+    inputsFile: Inputs;
+    sandboxInputs: Record<string, unknown>[];
+  }
+}
+
+export class SandboxFile extends BasePersistedFile {
+  private readonly workflowFile: WorkflowFile;
+  private readonly inputsFile: Inputs;
+  private readonly sandboxInputs: Record<string, unknown>[];
+
+  public constructor({
+    workflowContext,
+    workflowFile,
+    inputsFile,
+    sandboxInputs,
+  }: SandboxFile.Args) {
+    super({ workflowContext });
+    this.workflowFile = workflowFile;
+    this.inputsFile = inputsFile;
+    this.sandboxInputs = sandboxInputs;
+  }
+
+  protected getModulePath(): string[] {
+    return [this.workflowContext.moduleName, "sandbox"];
+  }
+
+  protected getFileStatements(): AstNode[] {
+    const sandboxRunnerField = python.field({
+      name: "runner",
+      initializer: python.instantiateClass({
+        classReference: python.reference({
+          name: "SandboxRunner",
+          modulePath:
+            this.workflowContext.sdkModulePathNames.SANDBOX_RUNNER_MODULE_PATH,
+        }),
+        arguments_: [
+          python.methodArgument({
+            name: "workflow",
+            value: python.reference({
+              name: "Workflow",
+              modulePath: this.workflowFile.getModulePath(),
+            }),
+          }),
+          python.methodArgument({
+            name: "inputs",
+            value: python.TypeInstantiation.list(
+              this.sandboxInputs.map((input) => this.getWorkflowInput(input))
+            ),
+          }),
+        ],
+      }),
+    });
+    this.inheritReferences(sandboxRunnerField);
+
+    return [
+      python.codeBlock(`\
+if __name__ != "__main__":
+    raise Exception("This file is not meant to be imported")
+        `),
+      sandboxRunnerField,
+      python.codeBlock("runner.run()"),
```

Comment:
You should be able to avoid a codeblock for this by using `python.invokeMethod`.